### PR TITLE
Categorie2

### PR DESCRIPTION
**Pull request by BenZen59**

### CODE REVIEW

**Review by Othmane-Khiter — DISMISSED** [review]
it's ok

File: src/components/Categories/Categories.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+
```
Comment (LoicBrassart) [inline src/components/Categories/Categories.css]: Useless empty line

File: src/components/Categories/Categories.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  flex-direction: column;
+}
+
+img {
+  width: 292px;
+  height: 349px;
+  border-top: 1px solid #000;
+  border-right: 1px solid #000;
+  border-left: 1px solid #000;
+
```
Comment (LoicBrassart) [inline src/components/Categories/Categories.css]: Useless empty line

File: src/components/FicheFilm/FicheFilm.css [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (LoicBrassart) [inline src/components/FicheFilm/FicheFilm.css:1]: Useless empty line

**Review by LoicBrassart — DISMISSED** [review]
Why do you ad these empty lines atop your files ?
And doesn't Prettier/ESLint rant about it ?

**Comment by LoicBrassart** [comment]
No SM validation

**Review by Othmane-Khiter — DISMISSED** [review]
it's ok no more empty lines

File: src/components/Categories/Categories.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      )
+      .then(({ data }) => {
+        setMovies(data.results);
+      });
+  }, []);
+
+  return (
+    <div>
+      <div>
+        {categories.map((categorie) => {
```
Comment (Hacene010) [inline src/components/Categories/Categories.jsx]: No french words (category in english)

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Need some changes

**Review by Othmane-Khiter — APPROVED** [review]
ok for "category" in english

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**